holoviews/plotting/mpl/plot.py

```diff
@@ -1097,7 +1100,8 @@ def initialize_plot(self):
         title_obj = None
         title = self._format_title(key)
         if self.show_title and len(self.coords) > 1 and title:
-            title_obj = self.handles['fig'].suptitle(title, **self._fontsize('title'))
+            title_obj = self.handles['fig'].suptitle(title, **self._fontsize('title'),
+                                                     y=self.suptitle_y)
```

Comment:
This is a syntax error, the y should precede the kwargs:

```suggestion
            title_obj = self.handles['fig'].suptitle(
                title, y=self.suptitle_y, **self._fontsize('title')
            )
```